packages/compass-telemetry/src/growth-experiments.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  useTrackOnChange,
+  type TrackFunction,
+} from '@mongodb-js/compass-telemetry/provider';
```

Comment:
CI is red because you're importing the package inside itself by absolute name, this should be a relative import